yeoreum/src/components/signUp/Form.tsx

```diff
@@ -0,0 +1,340 @@
+          ...pre,
+          validity: false,
+          message: '비밀번호를 먼저 입력해주세요.',
+        }));
+        return;
+      }
+      setUser(pre => ({ ...pre, passwordConfirm: current }));
+      if (user.password !== current) {
+        setPasswordConfirmInfo(pre => ({
+          ...pre,
+          validity: false,
+          message: '일치하지 않습니다.',
+        }));
+      } else {
+        setPasswordConfirmInfo(pre => ({
+          ...pre,
+          validity: true,
+          message: '일치합니다.',
+        }));
+      }
+    },
+    [passwordInfo.validity, user.password, user.passwordConfirm],
+  );
+  const onFocusPassword = useCallback(
+    (e: React.FocusEvent<HTMLInputElement>) => {
+      if (e.target.value.length !== 0) return;
+      setPasswordInfo(pre => ({
+        ...pre,
+        validity: undefined,
+        message: '특수문자, 영어, 숫자를 합친 6자 이상 14자 이하',
+      }));
+    },
+    [],
+  );
+  const onFocusPasswordConfirm = useCallback(
+    (e: React.FocusEvent<HTMLInputElement>) => {
+      if (e.target.value.length !== 0) return;
+      setPasswordConfirmInfo(pre => ({
+        ...pre,
+        validity: undefined,
+        message: '한 번 더 입력해주세요.',
+      }));
+    },
+    [],
+  );
+  return (
+    <Container onSubmit={e => e.preventDefault()}>
+      <Wrapper>
+        <Label>
+          <P>이메일</P>
+          <Input
+            type="email"
+            placeholder="이메일을 입력해주세요"
+            onChange={onChangeEmail}
+          />
+        </Label>
+        {mailStatus === 0 ? (
+          <Button onClick={onClickEmail}>인증하기</Button>
+        ) : (
+          <Button disabled={true}>전송됨</Button>
+        )}
+      </Wrapper>
+      {!emailInfo.validity && (
+        <AlertP success={emailInfo.validity}>{emailInfo.message}</AlertP>
+      )}
+      {mailStatus !== 0 ? (
+        <Wrapper>
+          <Label>
+            <P>00:00</P>
+            {mailStatus === 1 ? (
+              <Input
+                type="text"
+                maxLength={6}
+                onChange={e =>
+                  setUser(pre => ({ ...pre, emailCode: e.target.value }))
+                }
+                placeholder="인증코드"
+              />
+            ) : (
+              <Input
+                type="text"
+                maxLength={6}
+                placeholder="인증코드"
+                disabled
+              />
+            )}
+          </Label>
+          {mailStatus === 1 ? (
+            <Button onClick={onClickEmailCode}>인증하기</Button>
+          ) : null}
+        </Wrapper>
+      ) : null}
+      {emailCodeInfo.message.length > 0 && (
+        <AlertP success={emailCodeInfo.validity}>
+          {emailCodeInfo.message}
+        </AlertP>
+      )}
+      <Wrapper>
+        <Label>
+          <P>비밀번호</P>
+          <Input
+            onChange={onChangePassword}
+            onFocus={onFocusPassword}
+            type="password"
+            maxLength={14}
+            placeholder="비밀번호를 입력해주세요"
+          />
```

Comment:
```suggestion
          <Input
            name="password"
            onChange={onChangePassword}
            onFocus={onFocusPassword}
            type="password"
            maxLength={14}
            placeholder="비밀번호를 입력해주세요"
          />
```
- 만약에 위에 내 코드 아이디어를 참고한다면 이런 Input에다가 name을 추가해줘야 함!